google-chart.html

```diff
@@ -310,6 +311,33 @@
     _typeChanged: function() {
       this.$.loader.create(this.type, this.$.chartdiv)
           .then(function(chart) {
+
+            // get all gchart stylesheets
+            var stylesheets =
+                Polymer.dom(document.head)
+                    .querySelectorAll('link[rel="stylesheet"][type="text/css"]');
+            var gchartStylesheets = Array.from(stylesheets).filter(
```

Comment:
Seems simpler to just have one .forEach that does all this work instead of the filter/map/foreach, but your call if you prefer that style.